### PR TITLE
Configurable build

### PR DESCRIPTION
**Pull request by p-netm**
- Make steps used to build the app configurable.
- Add scenario where this scripts target opensrp-web, reveal

### CODE REVIEW

**Comment by moshthepitt** [comment]
> Add scenario where this scripts target opensrp-web, reveal

What differences necessitate this?

**Comment by p-netm** [comment]
> > Add scenario where this scripts target opensrp-web, reveal
> 
> What differences necessitate this?

@moshthepitt the only difference is in the build steps i.e.:
reveal only requires the build command, while opensrp web: might require transpiling the packages + building the app.
- arguably the web scenario would suffice

**Review by moshthepitt — APPROVED** [review]

File: defaults/main.yml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ---
+# react_build_cmds:
```
Comment (KipSigei) [inline defaults/main.yml:2]: remove commented out code

**Review by KipSigei — APPROVED** [review]

**Comment by p-netm** [comment]
@KipSigei the commented out section is intentional, shows the full structure of the `react_build_cmds` variable

**Pull request merged**